misc/pause-container/pause.c

```diff
@@ -0,0 +1,57 @@
+// This file is derived from the Kubernetes Project, Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Pedantically, Kubernetes is not Copyright Google.  The below copyright ("the kubernetes authors") is correct.